Software/WinIOT/Readme.MD

```diff
@@ -0,0 +1,23 @@
+# BrickPi3
+## Windows 10 IoT Core implementation for the BrickPi3.  These libraries can be run with the Raspberry Pi 2 or Raspberry Pi 3.
+
+The BrickPi3 was ported to C# [by Laurent Ellerbach](https://github.com/Ellerbach/).  There is a separate repository on Github with the code and [instructions on getting started here](https://github.com/Ellerbach/BrickPi3).
+
+Here are some shortcut for Windows 10 IoT Core with the BrickPi3.  
```

Comment:
Typo here (grammatically incorrect sentence).

Reply:
Got it, thank you for the detailed response!  I am going to leave the direct links in there and try to followup if they get broken, literally just to have something to say and to hope that Google picks it up.  

Made all the other suggestions though, which were great.  Good catch on the grammar, thanks!